doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+   NB-IoT is most suitable for static, low throughput applications like smart metering, smart agriculture, and smart city applications.
+   Also known as Cat-NB1.
+
+Near Field Communication (NFC)
+   A standards-based short-range wireless connectivity technology that enables two electronic devices to establish communication by bringing them close to each other.
+
+Network Co-Processor (NCP)
+   A co-processor offloading network functions from the host processor.
+   In the |NCS| context, it is typically used in OpenThread and Zigbee platform designs.
+
+Network Time Protocol (NTP)
+   A networking protocol for clock synchronization between computer systems over packet-switched, variable-latency data networks.
+
+NFC-A Listen Mode
+   The initial mode of an NFC Forum Device when it does not generate a carrier.
+   The device listens for the remote field of another device.
+   See Near Field Communication (NFC).
+
+Noise Factor (NF)
+   The relation of the Signal-to-Noise Ratio (SNR) in the device input to the SNR in the device output.
+
+Non-volatile Memory Controller (NVMC)
+   A controller used for writing and erasing the internal flash memory and the User Information Configuration Registers (UICR).
+
+nRF Connect for Cloud
+   Nordic Semiconductor's platform for connecting IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor chips, and more.
+   It includes a public REST API that can be used for building IoT solutions.
+   See `nRF Connect for Cloud`_.
+
+nRF repository
+   An |NCS| repository that does not have an externally maintained, open-source upstream.
+   It is exclusive to Nordic development.
+
+nRF Secure Immutable Bootloader (NSIB)
+   A bootloader created and maintained by Nordic Semiconductor that is built on Chain of Trust architecture.
+
+OpenAMP
+   A framework that provides software components that enable the development of software applications for Asymmetric Multiprocessing (AMP) systems.
+   See `OpenAMP`_.
+
+OpenThread
+   A portable and flexible `open-source implementation <OpenThread.io>`_ of the Thread networking protocol.
+
+OpenThread Border Router (OTBR)
+   A router that connects a Thread network to other IP-based networks, like Wi-Fi or Ethernet.
+   A Thread network requires a Border Router to connect to other networks.
+
+Operating System (OS)
+   A set of functions and data structures that manages system resources, hardware components, and the execution of programs and processes.
+   It is usually composed of a kernel, a scheduler, a file system, a memory manager, and other components.
+
+OSS repository
+   An |NCS| repository that tracks an upstream Open Source Software counterpart that is externally maintained.
+
+Over-the-Air (OTA)
+   Any type of wireless transmission.
+
+Patch
+   A method to export git commits for sharing them with others.
```

Comment:
This is also very often used indistinctly with "commit"

Reply:
A patch is not a commit; I think this should read 'A method to describe changes in one or more source code files.'

Reply:
fixed